src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -210,6 +211,12 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
             return Number.TryFormatUInt32(m_value, format, provider, destination, out charsWritten);
         }
 
+        /// <inheritdoc/>
+        bool IUtf8SpanFormattable.TryFormat(Span<byte> utf8Destination, out int bytesWritten, ReadOnlySpan<char> format, IFormatProvider? provider)
```

Comment:
Why are these all explicitly implemented... I thought we had them being publicly implemented where the existing types publicly exposed the same UTF-16 functionality?

Reply:
Did we agree to that? I'm ok with it personally, but I don't remember it being discussed.

Reply:
That was certainly the intent from the proposal. Having them be explicit just hides relatively "core" functionality and means they can't be a simple and better replacement for `Utf8Formatter`.

I don't see it explicitly called out in the proposal though, I just listed it as `Initial types that will implement the interface`.

If there are concerns, we can bubble it up first thing in API review tomorrow.

Reply:
> If there are concerns, we can bubble it up first thing in API review tomorrow.

Let's do that, to cross the `i`s and dot the `t`s as it were.  We can follow-up to make them all public subsequent to this PR if that's the decision.